src/test/java/uk/gov/companieshouse/fixtures/DissolutionFixtures.java

```diff
@@ -138,4 +140,33 @@ public static DirectorApproval generateDirectorApproval() {
 
         return approval;
     }
+
+    public static List<CompanyOfficerApi> generateCompanyOfficerList() {
+        final List<CompanyOfficerApi> officerList = new ArrayList<>();
```

Comment:
rather than declaring an array list, you can do `return Arrays.asList(officerOne, officerTwo);` at the end